### PR TITLE
Analytics: Valid named exports

### PR DESCRIPTION
**Pull request by sirreal**
Alternative to #20237

This was a quick and dirty implementation. It appears to work correctly in my smoke testing, however I'm not particularly happy with it. A more in depth refactor may need to be considered.

### CODE REVIEW

**Comment by ockham** [comment]
> This was a quick and dirty implementation. It appears to work correctly in my smoke testing, however I'm not particularly happy with it. A more in depth refactor may need to be considered.

Yeah, it is still kinda ugly. It just hasn't seen any love in ages. Would probably be good to move stuff to separate files. Want me to take it from here?

Some of this stuff is stateful, e.g. GA has an `initialized` flag, but I think what this PR does is okay and doesn't break it...

File: client/lib/analytics/index.js [hunk 1]
```diff
@@ -513,4 +295,228 @@ const analytics = {
 	},
 };
 emitter( analytics );
-export default analytics;
+
+const ga = {
```
Comment (ockham) [inline client/lib/analytics/index.js]: I'm a fan of inlined `export`s -- any reason for doing it [in bulk](https://github.com/Automattic/wp-calypso/pull/20247/files#diff-d088a91c46152a3ed739a47ed406c5bbR522)?

**Comment by ockham** [comment]
> Would probably be good to move stuff to separate files

To clarify, that way those object methods could become just plain functions at module level.

**Comment by ockham** [comment]
Oh, and if you'd rather unblock #20236, I think it's fine to go with #20237 for now.

**Comment by sirreal** [comment]
> Would probably be good to move stuff to separate files. Want me to take it from here?

There are a lot of improvements to be made here, but I don't want to take it on right now 🙂 

#20256 achieves the same goal with a much lighter touch 👍

File: client/lib/analytics/index.js (hunk 1 above)
Comment (sirreal) [reply client/lib/analytics/index.js]: No reason, I generally prefer inline myself. It's just the way it came out in this quick exploratory PR 😉

**Pull request closed without merging**